From 25db5a2e9567b5f5bb0f4b9f14c0625b324284d3 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Thu, 28 Nov 2024 11:09:23 +0100
Subject: [PATCH 749/841] fix(stm32mp2): remove warning in stm32_system_off

Remove a warning in stm32_system_off because it always occurs except
if the secondary core is manually unplug as the secondary core is
not unplug by default in Linux kernel shutdown sequence.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 49f826291448ce83eeb46eca5bad9cb779f60dcd fix(stm32mp2): remove warning in stm32_system_off]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I50a6520496148a8f3490124586dd4a20e88ab78e
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/421337
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
---
 plat/st/stm32mp2/stm32mp2_pm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/st/stm32mp2/stm32mp2_pm.c b/plat/st/stm32mp2/stm32mp2_pm.c
index 893f733cd..dd879e5db 100644
--- a/plat/st/stm32mp2/stm32mp2_pm.c
+++ b/plat/st/stm32mp2/stm32mp2_pm.c
@@ -904,7 +904,7 @@ static void __dead2 stm32_system_off(void)
 
 #if !STM32MP21
 	if (!stm32_freeze_other_core(core_id)) {
-		WARN("PSCI system off with other core running.\n");
+		VERBOSE("PSCI system off with other core running.\n");
 
 		/* Core is no more running */
 		stm32mp_state_set(STM32MP_SECONDARY_CPU, STATE_RUNNING, false);
-- 
2.47.2

